collectors/proc.plugin/sys_class_infiniband.c

```diff
@@ -14,39 +15,37 @@
+    FOREACH_COUNTER_PACKETS(GEN, __VA_ARGS__)                                                                          \
+    FOREACH_COUNTER_ERRORS(GEN, __VA_ARGS__)
+
+#define FOREACH_COUNTER_BYTES(GEN, ...)                                                                                \
+    GEN(port_rcv_data, bytes, "Received", 1, __VA_ARGS__)                                                              \
+    GEN(port_xmit_data, bytes, "Sent", -1, __VA_ARGS__)
+
+#define FOREACH_COUNTER_PACKETS(GEN, ...)                                                                              \
+    GEN(port_rcv_packets, packets, "Received", 1, __VA_ARGS__)                                                         \
+    GEN(port_xmit_packets, packets, "Sent", -1, __VA_ARGS__)                                                           \
+    GEN(multicast_rcv_packets, packets, "Mcast rcvd", 1, __VA_ARGS__)                                                  \
+    GEN(multicast_xmit_packets, packets, "Mcast sent", -1, __VA_ARGS__)                                                \
+    GEN(unicast_rcv_packets, packets, "Ucast rcvd", 1, __VA_ARGS__)                                                    \
+    GEN(unicast_xmit_packets, packets, "Ucast sent", -1, __VA_ARGS__)
+
+#define FOREACH_COUNTER_ERRORS(GEN, ...)                                                                               \
+    GEN(port_rcv_errors, errors, "Pkts malformated", 1, __VA_ARGS__)                                                   \
+    GEN(port_rcv_constraint_errors, errors, "Pkts rcvd discarded ", 1, __VA_ARGS__)                                    \
+    GEN(port_xmit_discards, errors, "Pkts sent discarded", 1, __VA_ARGS__)                                             \
+    GEN(port_xmit_wait, errors, "Tick Wait to send", 1, __VA_ARGS__)                                                   \
+    GEN(VL15_dropped, errors, "Pkts missed ressource", 1, __VA_ARGS__)                                                 \
+    GEN(excessive_buffer_overrun_errors, errors, "Buffer overrun", 1, __VA_ARGS__)                                     \
+    GEN(link_downed, errors, "Link Downed", 1, __VA_ARGS__)                                                            \
+    GEN(link_error_recovery, errors, "Link recovered", 1, __VA_ARGS__)                                                 \
+    GEN(local_link_integrity_errors, errors, "Link integrity err", 1, __VA_ARGS__)                                     \
+    GEN(symbol_error, errors, "Link minor errors", 1, __VA_ARGS__)                                                     \
+    GEN(port_rcv_remote_physical_errors, errors, "Pkts rcvd with EBP", 1, __VA_ARGS__)                                 \
+    GEN(port_rcv_switch_relay_errors, errors, "Pkts rcvd discarded by switch", 1, __VA_ARGS__)                         \
+    GEN(port_xmit_constraint_errors, errors, "Pkts sent discarded by switch", 1, __VA_ARGS__)
```

Comment:
Personally, I'm not fond of everything that our `.clang-format` does. I prefer to make columns aligned, as you did before.

Reply:
to be honest, when I saw the changes done by clang-format while I didn't commit before, I was so pissed... 
Just fixed it.